curriculum/outlines/Linear DS.md

```diff
@@ -0,0 +1,39 @@
+# Linear DS
+---
+## Outline
+1. Static Array
+	 1. 1D Static Array
+	 2. 2D Static Array
+	 3. 3D Static Array
+2. Dynamically-Resizeable Array
+3. BitSet
+4. Bitmasks (Basic bit manipulation)
+	  * Example: Flipping i-th bit in an integer.
+5. LinkedLists
```

Comment:
- important functions with their complexities
- pitfall: get function works in O(n)
- pitfall: Collections.sort(LinkedList); works in O(n log n) not O(n^2). refer to documentation or add its link